utils/yoke/Cargo.toml

```diff
@@ -4,7 +4,7 @@
 
 [package]
 name = "yoke"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
I did this as well because nobody uses `yoke-derive` directly so the way to force the new dependency to be pulled in is to pull in an updated yoke. Which seems fine by me.